front/pages/api/w/[wId]/labs/transcripts/default.ts

```diff
@@ -0,0 +1,83 @@
+import type { WithAPIErrorReponse } from "@dust-tt/types";
+import { assertNever } from "@dust-tt/types";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { Authenticator, getSession } from "@app/lib/auth";
+import { LabsTranscriptsConfigurationResource } from "@app/lib/resources/labs_transcripts_resource";
+import { apiError, withLogging } from "@app/logger/withlogging";
+import type { GetLabsTranscriptsConfigurationResponseBody } from "@app/pages/api/w/[wId]/labs/transcripts";
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<
+    WithAPIErrorReponse<GetLabsTranscriptsConfigurationResponseBody>
+  >
+): Promise<void> {
+  const session = await getSession(req, res);
+  const auth = await Authenticator.fromSession(
+    session,
+    req.query.wId as string
```

Comment:
Use a proper type assertion 🙏.

Reply:
You mean a typeguard on `req.query.wId`? 
Because that 'as string' is basically what we have everywhere else (search for `await Authenticator.fromSession(` and you'll see)

Reply:
Ok 😅! Even though it's not ideal.